pkg/scaffold/gopkgtoml.go

```diff
@@ -67,13 +69,11 @@ required = [
 
 [[override]]
   name = "sigs.k8s.io/controller-runtime"
-  revision = "v0.1.4"
+  version = "v0.1.4"
 
 [[constraint]]
   name = "github.com/operator-framework/operator-sdk"
-  # The version rule is used for a specific release and the master branch for in between releases.
-  branch = "master"
-  # version = "=v0.0.6"
+  version = "v0.0.6"
```

Comment:
@hasbro17 @AlexNPavel I made this change because I was not able to pull deps in a newly generated project while the SDK was locked to master. Any issue with this change?

Reply:
@estroz What happens if you change it to:
```TOML
branch = "refactor/controller-runtime"
# version = "v0.0.6"
```
We should have made that change before.

Reply:
Fixed by deleting local `dep` sources and rebuilding. I'll make the `branch` update and revert other changes.